core/src/main/java/bio/terra/pearl/core/model/search/SearchValueTypeDefinition.java

```diff
@@ -0,0 +1,22 @@
+package bio.terra.pearl.core.model.search;
+
+import bio.terra.pearl.core.model.survey.QuestionChoice;
+import bio.terra.pearl.core.service.search.terms.SearchValue;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@Setter
+@SuperBuilder
+@NoArgsConstructor
+@EqualsAndHashCode
+public class SearchValueTypeDefinition {
+    private SearchValue.SearchValueType type;
+    @Builder.Default
+    private List<QuestionChoice> choices = new ArrayList<>();
+    private boolean isMultiValued;
+    private boolean hasOtherQuestion;
```

Comment:
how about renaming these to match the SurveyQuestionDefintion fields?
```suggestion
    private boolean allowMultiple;
    private boolean allowOtherDescription;
```